tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "sourceMap": true,
     "moduleResolution": "node",
     "outDir": "build",
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Newer veresions of React (17 or 18+) do not require an explicit import of react in every JSX files anymore. This change is required otherwise Biome complains about every import of `react` for jsx components.